### PR TITLE
refactor(tests): refactor timeout, update snapshots (win32, chrome/firefox)

### PR DESCRIPTION
**Pull request by chalapkoStanislav**
Replaced all hardcoded timeouts with element-based waiting wherever feasible to improve test stability.
Updated snapshots with added masking.
Test fixes for daily run.

### CODE REVIEW

File: pages/dashboard/dashboard-page.js [hunk 1]
```diff
@@ -561,7 +561,7 @@ exports.DashboardPage = class DashboardPage extends BasePage {
   async moveFileToOtherTeamViaRightClick(fileName, otherTeamName) {
     const elem = this.page.locator(`button[title="${fileName}"]`).first();
     await elem.click({ button: 'right' });
-    await this.page.waitForTimeout(500);
+    await this.moveToFileMenuItem.waitFor({ state: 'visible' });
```
Comment (daniel-herrero) [inline pages/dashboard/dashboard-page.js:564]: I'd use `await expect(this.moveToFileMenuItem).toBeVisible();`, as it is basically the same but more legible.

File: pages/dashboard/team-page.js [hunk 2]
```diff
@@ -153,7 +153,7 @@ exports.TeamPage = class TeamPage extends BasePage {
   }
 
   async isTeamDeleted(teamName) {
-    await this.page.waitForTimeout(5000)
+    await this.page.getByText('Your Penpot').waitFor({ state: 'visible' , timeout: 8000});
```
Comment (daniel-herrero) [inline pages/dashboard/team-page.js:156]: I'd also use the `toBeVisible` assertion:
`await expect(this.page.getByText('Your Penpot')).toBeVisible({timeout: 8000});`

File: pages/workspace/design-panel-page.js [hunk 3]
```diff
@@ -1090,9 +1090,8 @@ exports.DesignPanelPage = class DesignPanelPage extends BasePage {
   }
 
   async isAnnotationAddedToComponent(value) {
-    await this.page.waitForTimeout(500);
+    await this.annotationCreateTitle.waitFor({ state : 'visible' });
```
Comment (daniel-herrero) [inline pages/workspace/design-panel-page.js:1093]: I'd change it in favor of the locator toBeVisible assertion, as in the two previous comments.

File: pages/workspace/prototype-panel-page.js [hunk 4]
```diff
@@ -35,7 +35,7 @@ exports.PrototypePanelPage = class PrototypePanelPage extends BasePage {
   }
 
   async dragAndDropPrototypeArrowConnector(x, y) {
-    await this.page.waitForTimeout(200);
+    await this.prototypeArrowConnector.waitFor({ state: 'visible' });
```
Comment (daniel-herrero) [inline pages/workspace/prototype-panel-page.js:38]: Same here

**Review by daniel-herrero — CHANGES_REQUESTED** [review]
Please, consider to use the locator `toBeVisible` assertion instead of the `waitFor({ state: 'visible' }`

File: pages/dashboard/dashboard-page.js (hunk 1 above)
Comment (chalapkoStanislav) [reply pages/dashboard/dashboard-page.js]: done

File: pages/dashboard/team-page.js (hunk 2 above)
Comment (chalapkoStanislav) [reply pages/dashboard/team-page.js]: done

File: pages/workspace/design-panel-page.js (hunk 3 above)
Comment (chalapkoStanislav) [reply pages/workspace/design-panel-page.js]: done

File: pages/workspace/prototype-panel-page.js (hunk 4 above)
Comment (chalapkoStanislav) [reply pages/workspace/prototype-panel-page.js]: done

**Review by daniel-herrero — APPROVED** [review]
Perfect! Don't forget to "Squash and merge" and delete the remote branch.

**Pull request merged**